client/modules/IDE/components/Console.jsx

```diff
@@ -48,8 +85,23 @@ class Console extends React.Component {
               );
             }
             return (
-              <div key={consoleEvent.id} className={`preview-console__${method}`}>
-                {Object.keys(args).map(key => <span key={`${consoleEvent.id}-${key}`}>{args[key]}</span>)}
+              <div key={consoleEvent.id} className={`preview-console__${method} theme-with-${theme}-background`}>
+                { times > 1 &&
+                  <div className={`preview-console__logged-times preview-console__${method}__logged-times`}>{times}</div>
```

Comment:
you should only need one class here: `.preview-console__logged-times`. i'll explain below how to adjust the SCSS.